app/gui/project-manager-shim-middleware/index.ts

```diff
@@ -147,12 +147,18 @@ export default function projectManagerShimMiddleware(
   response: http.ServerResponse,
   next: () => void,
 ) {
-  const requestUrl = request.url
-  const requestPath = requestUrl?.split('?')[0]?.split('#')[0]
-  if (requestUrl != null && requestUrl.startsWith('/api/project-manager/')) {
-    const actualUrl = new URL(
-      requestUrl.replace(/^\/api\/project-manager/, GLOBAL_CONFIG.projectManagerHttpEndpoint),
+  const requestUrl = request.url ?? ''
+  if (!requestUrl.startsWith('/api/')) return next()
+  console.log('requestUrl', requestUrl)
+  const url = new URL(requestUrl, 'https://apishim.local')
+  const requestPath = url.pathname
+  if (requestPath.startsWith('/api/project-manager/')) {
+    const urlString = requestUrl.replace(
+      /^\/api\/project-manager/,
+      GLOBAL_CONFIG.projectManagerHttpEndpoint,
     )
+    console.log('urlString', urlString)
```

Comment:
Stray logs - or we want to keep them?

If not, use `console.debug` in the future - these are caught by linter